src/help/zaphelp/contents/releases/2_6_0.html

```diff
@@ -0,0 +1,235 @@
+<li>Issue 2415 : Show the reason why an active scanner was skipped</li>
+<li>Issue 2559 : Do not clean up unsaved (file based) sessions</li>
+<li>Issue 2570 : Change proxy option to remove unsupported encoding</li>
+<li>Issue 2592 : Differentiate the source of alerts</li>
+<li>Issue 2611 : Change HTTP breakpoint dialogues to modal</li>
+<li>Issue 2633 : Enhance Client Cert File Chooser</li>
+<li>Issue 2647 : Support a/pscan rule configuration</li>
+<li>Issue 2655 : Provide skip reason for Script Active Scan Rules</li>
+<li>Issue 2682 : Sort (main) help add-on TOC entries</li>
+<li>Issue 2690 : Support ignoring specified forms when checking for CSRF vulnerabilities</li>
+<li>Issue 2701 : Enhancement Request: Factory Reset</li>
+<li>Issue 2723 : Support POST requests for API actions</li>
+<li>Issue 2728 : Allow to remove spider parsers and filters</li>
+<li>Issue 2750 : Add a reasonably strong CSP to the API</li>
+<li>Issue 2773 : Use UTF-8 to read/write ZAP scripts</li>
+<li>Issue 2782 : Support the -configfile cmdline parameter</li>
+<li>Issue 2825 : Additional Commentary for JS templates</li>
+<li>Issue 2853 : Override Alert details</li>
+<li>Issue 2855 : Support break functionality in the API</li>
+<li>Issue 2865 : Normalise Include/Exclude context panels</li>
+<li>Issue 2886 : Option to generate reports in Markdown format</li>
+<li>Issue 2891 : Show the cause why a script was not loaded</li>
+<li>Issue 2936 : Always set Java mem to 1/4 available (over 512Mb)</li>
+<li>Issue 2937 : Change ZAP API to read/use the request body</li>
+<li>Issue 2939 : Use non absolute URI base HTML element in spider</li>
+<li>Issue 2951 : Support active scan rule and scan max duration</li>
+<li>Issue 2954 : Allow to export a Context through the context menu</li>
+<li>Issue 2966 : Use L&F specified through JVM args</li>
+<li>Issue 2970 : Allow to configure, by script type, the enabled state of new/loaded scripts</li>
+<li>Issue 2982 : Allow to disable default standard output logging</li>
+<li>Issue 2994 : show column 'Size Resp. Body' of history in bytes</li>
+<li>Issue 3004 : Allow to passive scan just HTTP messages in scope</li>
+<li>Issue 3038 : Return request's type through the ZAP API</li>
+<li>Issue 3042 : Allow to select multiple parameters in Params tab</li>
+<li>Issue 3050 : Return requests' timestamp/RTT through the ZAP API</li>
+<li>Issue 3061 : Allow to deprecate API endpoints</li>
+<li>Issue 3079 : Added cookie ignore list rule and inc sleep default to 20 to reduce FPs</li>
+<li>Issue 3081 : Change default time to 15 and make publicly accessible</li>
+<li>Issue 3098 : Log to file even if ZAP is run 'inline'</li>
+<li>Issue 3118 : include subjectAlternativeName extension in generated certificates</li>
+</ul>
+
+<H2>Bug fixes:</H2>
+<ul>
+<li>Issue 581 : Transaction rollback: serialization failure</li>
+<li>Issue 649 : AddOnLoader suppresses exceptions occurring during the initialization of extensions</li>
+<li>Issue 1064 : Dirbuster console errors</li>
```

Comment:
Same here (although if indeed we want to keep it close we should move it to another release page).